docker-compose.yml

```diff
@@ -150,6 +150,7 @@ services:
       - UKEF_INTERNAL_NOTIFICATION=test
       - NODE_ENV
       - RATE_LIMIT_THRESHOLD
+      - CHANGE_STREAM_ENABLED
```

Comment:
is this always false as if it's true we cant use DELETION_AUDIT_LOGS_DELETE_AFTER_SECONDS as its not passed in and thatll throw an error when deleting a deal?

Reply:
The change stream can't be turned on if not in a replica set so I haven't passed in the new env var here